### DISCUSSION TITLE
Sensor not starting anymore with used C... transmitter

### DISCUSSION THREAD

**Discussion by Raminta-ltu** [NightscoutFoundation/xDrip#2973] [Q&A] [answered]
I've got C... transmitter which worked fine for 60 days and then something happened that it was no longer connecting.

Tried to reniew Xdrip to the latest version and to choose the right settings. But it didn't help... Also tried with new sensor. Didn't work.

When I try to start sensor, I get notification that something is wrong with time or other settings. But I selected everything according to instructions. So sensor doesn't start. 

Maybe you could send me somehow a file with settings which I could apload to xdrip app?

![inbound2369862918834864498](https://github.com/NightscoutFoundation/xDrip/assets/140712144/8f0b6928-ba52-4902-b200-b5cb45e9a1c2)

![inbound2288077338817096488](https://github.com/NightscoutFoundation/xDrip/assets/140712144/4385eb07-7f6b-4f34-8c9f-25aba96cb380)

**Comment by Navid200** [comment]
What is the phone brand and model?

**Reply by Raminta-ltu** [reply]
Xiaomi Redmi Note 9 pro

**Comment by jamorham** [comment]
Please enable ob1 debugging as described here. https://github.com/NightscoutFoundation/xDrip/wiki/Enabling-Debugging-for-OB1-collector try again and if it still isn't working please submit the logs with an identifier so we know its you and I can take a look. We recently added support to detected failed transmitters as the Cxxxx ones seem to fail more frequently but I don't see that flagged on your status page.

**Reply by Raminta-ltu** [reply]
Uploaded logs with email name - [email name removed]

**Comment by Navid200** [comment]
@Raminta-ltu These are the instructions for uploading logs: https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

**Reply by Raminta-ltu** [reply]
Uploaded logs with my email name[email removed]. I hope I did it right 😅

**Reply by jamorham** [reply, marked as answer]
As far as I can tell the logs show the transmitter has failed but what I don't yet understand is why you're not getting the warning on the status page about this.

**Reply by Raminta-ltu** [reply]
Meaning - this transmitter is not usable anymore? I need new one?

**Reply by jamorham** [reply]
Yes, that is my best guess based on what the logs show

**Reply by Raminta-ltu** [reply]
I called dexcom and they are sending new transmitter.

Thanks for the help :)

**Comment by Navid200** [comment]
@Raminta-ltu It's not easy to make a diagnosis remotely with 100% certainty.

Do you happen to have a phone other than Samsung and Xiaomi yourself or one you can borrow?
It only needs to be Android 6 or newer.
